crates/turborepo-lib/src/commands/info.rs

```diff
@@ -14,7 +16,15 @@ pub fn run(base: &mut CommandBase, workspace: Option<&str>) -> Result<()> {
     let package_manager =
         PackageManager::get_package_manager(&base.repo_root, Some(&root_package_json))?;
 
-    let package_graph = PackageGraph::builder(&base.repo_root, root_package_json)
+    let opts: Opts = base.args().try_into()?;
+
+    let root_turbo_json = TurboJson::load(
+        &base.repo_root,
+        &root_package_json,
+        opts.run_opts.single_package,
+    )?;
```

Comment:
Yeah it was a dumb reason, I just didn't want to clone the `root_package_json` since `PackageGraph` takes ownership. I'll clone it for now, and maybe we can add a way to get a reference from package_graph sometime.